README.md

```diff
@@ -917,6 +915,30 @@ job.depends([job1, job2]);
 // Clear any dependencies previously added on this job
 job.depends();
 ```
+Added antecendent jobs must have already had `.save()` run on them, so they will have the `_id` attribute that
+is used to form the dependency. In practice, that often means that the dependent job will be created within the `save()` callback of its antecedent:
+
```

Comment:
Many / most job-collection users are writing workers within the Meteor server environment where the use of Fibers makes callbacks largely unnecessary. (As an aside, the npm package [also supports using FIbers](https://www.npmjs.com/package/meteor-job#job-setddp-ddp-collectionname-fiber) if you are so inclined.

Because most users won't write workers in an environment that requires callbacks, I prefer to "Fiber blocking" code in the examples because it is simpler and closer to what most users will actually write...